### ISSUE TITLE
output incompatible with i18next 2.0

### ISSUE THREAD

**Issue by VilmaRamos** [i18next/i18next-gettext-converter#32]
this plugin doesn't work with the new version of i18next. For instance, in the current version multiple plural forms are parsed like this:
```json
{           
  "ar": { 
    "translation": {  
      "key": "singular",
      "key_plural_0": "zero",
      "key_plural_2": "two",
      "key_plural_3": "few",
      "key_plural_11": "many",
      "key_plural_100": "plural"
    } 
  }
};
```

but i18n 2.0 expects something like this:
```json
{
  "ar": {
    "translation": {
      "key_0": "zero",
      "key_1": "singular",
      "key_2": "two",
      "key_3": "few",
      "key_11": "many",
      "key_100": "plural"
    }
  }
}
```

**Comment by jamuhl** [comment]
yes...you're right. Not yet had the time to get this done. I hope to get this done when v2 gets released. Should be to hard to change. PR would be very welcome.

**Comment by jamuhl** [comment]
https://github.com/i18next/i18next-gettext-converter/commit/593940bcb512a9486cd6594847b47ca2cde033a5 should change it to be compatible with i18next v2 format.

`npm install -g i18next-conv` (latest 2.0.0)

**Comment by jamuhl** [comment]
still wrong for json -> po

**Comment by jamuhl** [comment]
see: https://github.com/i18next/i18next/issues/468

**Comment by jamuhl** [comment]
should be finally fixed in 2.0.1
